packages/cozy-harvest-lib/src/services/bi-http.js

```diff
@@ -73,6 +73,20 @@ const encodeToForm = rawForm => {
   return formData
 }
 
+export const getBIUserConfig = async (config, biAccessToken) => {
+  return await biRequest('GET', '/users/me/config', config, null, biAccessToken)
```

Comment:
nit: having on object for `config / null / biAccessToken` can be easier to maintain and understand

Reply:
Agreed, will do it next time I need to add something, I am not very happy with this lib.